stable/artifactory/templates/artifactory-statefulset.yaml

```diff
@@ -238,7 +255,7 @@ spec:
       {{- $tag := .image.tag }}
       {{- $mountPath := $.Values.artifactory.persistence.mountPath }}
       {{- range .names }}
-      - name: {{ . | replace "_" "-" }}
+      - name: {{ . | replace "_" "-" }}-logger
```

Comment:
Can we do the fixes/changes to the loggers in a different PR. Just to keep it tidy.